compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1467,7 +1473,11 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
             return;
         }
 
-        let unused_macro = self.unused_macros.iter().find_map(|(def_id, (_, unused_ident))| {
+        // Make ordering consistent before iteration
+        #[allow(rustc::potential_query_instability)]
+        let mut unused_macros: Vec<_> = self.unused_macros.iter().collect();
```

Comment:
You could use `UnordMap` for `unused_macros`, it already has methods for producing sorted vectors.

Reply:
I think it cannot be directly sorted using UnordMap because `LocalDefId` is not implementing `StableCompare` or I just couldn't achieve that.

Reply:
Okay, let's keep explicit sorting then.